packages/typespec-ts/src/lib.ts

```diff
@@ -11,6 +11,7 @@ import { Options } from "prettier";
 
 export interface EmitterOptions extends RLCOptions {
   branded?: boolean;
+  generateOrphanModels?: boolean;
```

Comment:
Feel like it's better to put it as an emitter layer option instead of a rlc option.